### PR TITLE
Feature/kitosudv 2219 task2381

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs [hunk 1]
```diff
@@ -151,11 +156,12 @@ public void Can_Map_General_Data_Properties()
             var input = A<GeneralDataWriteRequestDTO>();
 
             //Act
-            var output = _sut.MapGeneralData(input);
+            var output = _sut.FromPOST(new CreateItSystemUsageRequestDTO() { General = input });
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs:159]: Hvorfor FromPOST og ikke FromPATCH her?

File: Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs [hunk 2]
```diff
@@ -166,12 +172,13 @@ public void Map_General_Data_Properties_Resets_Validity_If_SourceValue_Not_Defin
             input.Validity = null;
 
             //Act
-            var output = _sut.MapGeneralData(input);
+            var output = _sut.FromPOST(new CreateItSystemUsageRequestDTO() { General = input });
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs:175]: Hvorfor FromPOST og ikke FromPATCH her?

File: Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs [hunk 3]
```diff
@@ -182,10 +189,11 @@ public void Map_General_Data_Properties_Resets_NumberOfExpectedUsers_If_SourceVa
             input.NumberOfExpectedUsers = null;
 
             //Act
-            var output = _sut.MapGeneralData(input);
+            var output = _sut.FromPOST(new CreateItSystemUsageRequestDTO() { General = input });
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs:192]: Hvorfor FromPOST og ikke FromPATCH her?

File: Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs [hunk 4]
```diff
@@ -196,10 +204,11 @@ public void Map_General_Data_Properties_Resets_AssociatedProjects_If_SourceValue
             input.AssociatedProjectUuids = null;
 
             //Act
-            var output = _sut.MapGeneralData(input);
+            var output = _sut.FromPOST(new CreateItSystemUsageRequestDTO() { General = input });
```
Comment (JacobMalling) [inline Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs:207]: Hvorfor FromPOST og ikke FromPATCH her?

**Review by JacobMalling — APPROVED** [review]
En enkelt ting i unit testen som går igen et par gange

File: Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs (hunk 1 above)
Comment (mrjsawdk) [reply Tests.Unit.Presentation.Web/Models/V2/ItSystemUsageWriteModelMapperTest.cs:159]: lige her er det meningen (se assertion)

**Pull request merged**